src/shared/utils.tsx

```diff
@@ -0,0 +1,22 @@
+import moment from "moment";
+
+class Utils {
+
+    static formatDate(dateStr: string): string {
+        const date = new Date(dateStr);
+
+        return moment(date).format("YYYY MM DD");
+    }
+
+    static formatGender(gender: string): string {
+        return gender.includes('F') ? 'Female' : 'Male';
```

Comment:
Please, use double quotemarks over single quotes.

```suggestion
        return gender.includes("F") ? "Female" : "Male";
```